### PR TITLE
RSDK-4446 Unify CLI arguments into single style

### PR DESCRIPTION
**Pull request by benjirewis**
RSDK-4446

Moves all CLI string flags to a `const` block at the top of `cli/app.go`. Modifies all usages of `c.String` and `c.Bool` to use these constants.

### CODE REVIEW

File: cli/module_registry.go [hunk 1]
```diff
@@ -176,12 +176,12 @@ func UpdateModuleAction(c *cli.Context) error {
...
-	orgIDArg := c.String("org-id")
-	nameArg := c.String("name")
-	versionArg := c.String("version")
-	platformArg := c.String("platform")
+	manifestPathArg := c.String(moduleFlagPath)
+	publicNamespaceArg := c.String(moduleFlagPublicNamespace)
+	orgIDArg := c.String(moduleFlagOrgID)
+	nameArg := c.String(moduleFlagName)
+	versionArg := c.String(moduleFlagVersion)
+	platformArg := c.String(moduleFlagVersion)
```
Comment (zaporter-work) [inline cli/module_registry.go:184]:
```suggestion
	platformArg := c.String(moduleFlagPlatform)
```

**Review by zaporter-work — APPROVED** [review]
LGTM!

This is a good change.

**Review by cheukt — APPROVED** [review]
LGTM

File: cli/module_registry.go (hunk 1 above)
Comment (benjirewis) [reply cli/module_registry.go]: 😮‍💨 thanks for catching. Looked through one more time to double check I didn't mess up any others...

**Pull request merged**